rwiki/rwiki-tool/tool/src/webapp/WEB-INF/vm/full_search.vm

```diff
@@ -77,7 +77,7 @@
 				<!--is-->
 				<li>
 					<h5> <a href="${result.url}" class="searchTopLink" >${util.escapeHtml($searchBean.pageNameFromSearchTitle(${result.title}))}</a></h5> 
-					<div class="searchItemBody" >${util.escapeHtml($result.searchResult)}</div>
```

Comment:
it's possible that escaping is to restrictive but since your displaying user supplied content it shouldn't be trusted either. I would suggest using formatted text if you don't want to escape.

Reply:
Used `escapeHtmlFormattedText`